examples/Flight1v2/Flight1v2.ino

```diff
@@ -461,18 +461,35 @@ void configCB(BLEDevice unused1, BLECharacteristic unused2)
   uint8_t data = buf[5];
   uint8_t signalEvents [4] = {0x00, 0x00, 0x04, 0x00};
 
-  if (!memcmp(buf, signalEvents, 4) && data) {
-    enableAllFunc();
+  if (!memcmp(buf, signalEvents, 4)) {
     switch (command) {
-      case 'o':
-        Flight1.shortMode = true;
+      case 'm':
+        data ? enableAllFunc() : disableAllFunc();
         break;
-      default:
-        Flight1.shortMode = false;
+      case 'f':
+        data ? AccGyr.Enable_Free_Fall_Detection(LSM6DSO_INT1_PIN) : AccGyr.Disable_Free_Fall_Detection();
+        break;
+      case 'd':
+        data ? AccGyr.Enable_Double_Tap_Detection(LSM6DSO_INT1_PIN) : AccGyr.Disable_Double_Tap_Detection();
+        break;
+      case 's':
+        data ? AccGyr.Enable_Single_Tap_Detection(LSM6DSO_INT1_PIN) : AccGyr.Disable_Single_Tap_Detection();
+        break;
+      case 'p':
+        data ? AccGyr.Enable_Pedometer() : AccGyr.Disable_Pedometer();
+        data ? AccGyr.Step_Counter_Reset() : false;
```

Comment:
Replace "data ? AccGyr.Step_Counter_Reset() : false;" with "data ? AccGyr.Step_Counter_Reset() : 0;" to avoid a compilation warning

Reply:
Warning fixed in commit 512eaf8